docs/source/getting_started/index.rst

```diff
@@ -1,10 +1,26 @@
 Getting started
-#######################################
+##############################################################################
 
-The following pages take you step by step with getting started with the CS Unplugged project. Start here if you’re new to the project.
+Welcome to the CS Unplugged community!
+We would love for you to get involved into making this project as great as
+possible!
+This documentation will help you understand how the project is setup,
+basic steps on how to use it, and our expectations of your contributions.
+
+Firstly, this project adheres to the Contributor Covenant code of conduct.
+By participating, you are expected to uphold this code.
+Read our :doc:`Code of Conduct <code_of_conduct>` before continuing.
+
```

Comment:
The majority of this is word-for-word match with Kordac docs, so don't really know how you want me to improve it.

I've moved it inside of a `note` block, added `please`, and added the email for quick reference.